src/commcare_cloud/ansible/roles/lpar2rrd/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
-- include: install_agent.yml
+- include_tasks: install_agent.yml
   tags: lpar2rrd_agent
```

Comment:
The first task (install_agent.yml) is a static task because it is included unconditionally and does not depend on any dynamic conditions or variables.